docs/src/pages/guides/styling.md

```diff
@@ -159,6 +159,15 @@ Now you're ready to write Tailwind! Our recommended approach is to create a `src
 
 As an alternative to `src/styles/global.css`, You may also add Tailwind utilities to individual `pages/*.astro` components in `<style>` tags, but be mindful of duplication! If you end up creating multiple Tailwind-managed stylesheets for your site, make sure you're not sending the same CSS to users over and over again in separate CSS files.
 
+#### v20 Tweaks to the tailwind import
+As of [Version 20](https://github.com/snowpackjs/astro/releases/tag/astro%400.20.0), Astro will no longer bundle, build and process public files. You now need to move your styles into the `src` directory and import them like so, making use of [Astro Resolve](http://localhost:3000/reference/api-reference#astroresolve): 
+```html
```

Comment:
```suggestion
```astro
```
can you change the language to astro rather than html, especially since we're using astro-specific syntax? otherwise, looks great to me!

Reply:
Heck that worked 👀
How does markdown know what `astro` is in this case? Where is that defined? 

![CleanShot 2021-08-30 at 19 29 08](https://user-images.githubusercontent.com/16897508/131318342-2ca5956d-36d1-4cdb-83ae-182592b8632a.png)

Reply:
The built-in highlighting in astro is powered by Prism atm, and there's an astro plugin that was built to add astro language support to Prism! (Github also has syntax highlighting for astro because we made a pr to https://github.com/github/linguist